src/test/scala/scala/concurrent/stm/IsolatedRefSuite.scala

```diff
@@ -39,32 +41,32 @@ class IsolatedRefSuite extends FunSuite {
   // This implements Ref.View, but requires a surrounding InTxn context and
   // forwards to Ref's methods.
   class DynamicView[A](val ref: Ref[A]) extends Ref.View[A] {
-    implicit def txn = Txn.findCurrent.get
```

Comment:
Again, this "worked" only without type annotation; make sure we don't create infinite loop here.